scripts/installer.py

```diff
@@ -11,6 +11,19 @@
 from time import sleep
 import sys
 
+
+def is_arm():
+    try:
+        with open('/proc/cpuinfo', 'r') as cpuinfo:
+            for line in cpuinfo:
+                if line.startswith('model name'):
+                    hardware = line.split(':')[1].strip()
+                    if 'ARM' in hardware:
+                        return True
+    except FileNotFoundError:
+        pass
+    return False
+
```

Comment:
This might not work on every arm cpu, try using `platform.machine()` or if that fails then call the command `uname -m` to find architecture.